src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/PortableInfobox.java

```diff
@@ -172,6 +172,12 @@ public String getDataImageName() {
     return imageTag.getAttribute("data-image-name");
   }
 
+  public PortableInfobox waitForUrlToContain(String expectedURL){
```

Comment:
This seems like it should be a part of a page functionality, instead of an infobox.